apps/mobile/src/store/storage.ts

```diff
@@ -1,19 +1,69 @@
+/* eslint-disable @typescript-eslint/no-extraneous-class */
 import { Storage } from 'redux-persist'
 import { MMKV } from 'react-native-mmkv'
+import { mapStorageTypeToIds, STORAGE_IDS, STORAGE_TYPES } from './constants'
 
-const storage = new MMKV()
+const safeStorage = new MMKV({
+  id: STORAGE_IDS.SAFE,
+})
 
 export const reduxStorage: Storage = {
```

Comment:
Btw, have you seen that we already have a very similar abstraction here: https://github.com/safe-global/safe-wallet-monorepo/blob/dev/apps/web/src/services/local-storage/Storage.ts

Perhaps `redux-persist` could be another "backend" for that storage class. Currently it only supports localStorage and sessionStorage.